addons/mail/static/src/components/chatter_topbar/chatter_topbar.xml

```diff
@@ -37,10 +37,18 @@
                     </t>
                     <div class="o-autogrow"/>
                         <div class="o_ChatterTopbar_rightSection">
-                            <button class="btn btn-link o_ChatterTopbar_button o_ChatterTopbar_buttonAttachments" type="button" t-att-disabled="chatter.isDisabled" t-on-click="_onClickAttachments">
+                            <t t-set="attachmentCount" t-value="chatter.thread ? chatter.thread.allAttachments.length : 0"/>
+                            <t t-set="attachmentButtonAria">%s Attachment(s)</t>
+                            <button class="btn btn-link o_ChatterTopbar_button o_ChatterTopbar_buttonAttachments"
```

Comment:
keep single line, avoid extra diff

Reply:
Done, although I would've preferred several lines because there are a lot of attributes here.